### PR TITLE
[Gallery] Update README with steps to create a release

### PR DESCRIPTION
**Pull request by rami-a**
Update README with steps to create a release

### CODE REVIEW

**Comment by rami-a** [comment]
@RedBrogdon could you take a look at this when you get a chance. I want any one within the Flutter org to be able to create a gallery release.

File: gallery/README.md [hunk 1]
```diff
@@ -72,3 +72,28 @@ more details):
...
+    * `flutter pub global activate peanut`
+    * `flutter pub global run peanut:peanut`
+    * `git push upstream gh-pages:gh-pages` (You may have to force push)
+   
+4. Publish the android release (using the correct signing certificates)
+    * Create the app bundle with `flutter build appbundle`
+    * Upload to the play store console
+    * Publish the play store release
+    * Create the APK with `flutter build apk` (this is for the Github release)
+
```
Comment (perclasson) [inline gallery/README.md]: Period at the end?

File: gallery/README.md [hunk 2]
```diff
@@ -72,3 +72,28 @@ more details):
 ```
 flutter pub run grinder update-code-segments
 ```
+
+## Creating a new release (for Flutter org members)
+
+1. Bump the version number up in the `pubspec.yaml`. Use semantic versioning to determine 
+   which number to increment. For example `2.2.0+020200` should become `2.3.0+020300`.
+   
+2. Create a tag on the `master` branch in the form of `gallery-v2.3`.
```
Comment (perclasson) [inline gallery/README.md]: Do we want to add the git command for this?

File: gallery/README.md [hunk 3]
```diff
@@ -72,3 +72,28 @@ more details):
...
+    * Create the app bundle with `flutter build appbundle`
+    * Upload to the play store console
+    * Publish the play store release
+    * Create the APK with `flutter build apk` (this is for the Github release)
+
+5. Draft a release in Github from the tag you created, call the release `Flutter Gallery 2.3`
+    * Upload the Android APK from above
+    * Create and upload the macOS build by running `flutter build macos` and zipping the 
+      app inside `build/macos/Build/Products/Release`.
+    * Optional: Create and upload the linux/windows builds
```
Comment (perclasson) [inline gallery/README.md]: super nit: Linux/Windows

File: gallery/README.md (hunk 1 above)
Comment (rami-a) [reply gallery/README.md]: done

File: gallery/README.md (hunk 2 above)
Comment (rami-a) [reply gallery/README.md]: done

File: gallery/README.md (hunk 3 above)
Comment (rami-a) [reply gallery/README.md]: done

File: gallery/README.md [hunk 4]
```diff
@@ -72,3 +72,30 @@ more details):
...
+## Creating a new release (for Flutter org members)
+
+1. Bump the version number up in the `pubspec.yaml`. Use semantic versioning to determine 
+   which number to increment. For example `2.2.0+020200` should become `2.3.0+020300`.
+   
+2. Create a tag on the `master` branch of this repo in the form of `gallery-v2.3`.
+	* `git tag gallery-v2.3`
+	* `git push --tags`
+
+3. Publish the web release (using the peanut package).
```
Comment (RedBrogdon) [inline gallery/README.md]: Consider linkifying the name "peanut" to point to https://pub.dev/packages/peanut.

File: gallery/README.md [hunk 5]
```diff
@@ -72,3 +72,30 @@ more details):
...
+    * `flutter pub global activate peanut`
+    * `flutter pub global run peanut:peanut`
+    * `git push upstream gh-pages:gh-pages` (You may have to force push)
```
Comment (RedBrogdon) [inline gallery/README.md]: @kevmoo, you had some thoughts on force-pushing gh-pages, didn't you?

File: gallery/README.md [hunk 6]
```diff
@@ -72,3 +72,30 @@ more details):
...
+   
+4. Publish the android release (using the correct signing certificates)
```
Comment (RedBrogdon) [inline gallery/README.md]: nit: Capitalize "Android."

File: gallery/README.md [hunk 7]
```diff
@@ -72,3 +72,30 @@ more details):
...
+    * Create the app bundle with `flutter build appbundle`.
+    * Upload to the play store console.
```
Comment (RedBrogdon) [inline gallery/README.md]: nit: Capitalize "Play" ("store" is correct as lower case).

File: gallery/README.md (hunk 5 above)
Comment (kevmoo) [reply gallery/README.md]: Don't force push!

Instead `git update-ref refs/heads/gh-pages origin/gh-pages` – this updates your local `gh-pages` to align w/ remote!

**Pull request merged**